app/aem/core/src/test/resources/foreach.apm

```diff
@@ -23,3 +23,23 @@ FOR-EACH i IN ['a', 'b'] BEGIN
     SHOW $i+'/'+$j
   END
 END
+
+FOR-EACH [i, j] IN [['a', 'b'], ['c', 'd']] BEGIN

```

Comment:
This ([i, j]) looks like creation of array with 2 elements. It would be simpler by using just FOR i, j IN. Why did you use 2 dimensional array in second part of statement? In this contex it is hard to tell What would be assigned to i and j. Cannot you just stay with flat, one dimensional array? In My opinion syntax like FOR i, j IN ['a', 'b', 'c', 'd'] would be simpler, and better. Users can take advantage of multiline array definitione to organize the data legibly. For example:
DEFINE users ['user@email', 'name', 'password',
'user2@email', 'name2', 'password',
... 
'usern@email', 'namen', 'password']

Reply:
this one-liner is only an example, you can write it in other ways:
```
FOR-EACH [i, j] 
IN [
    ['a', 'b'], 
    ['c', 'd']
] 
BEGIN
```
or
```
DEFINE tabs [
    ['a', 'b'], 
    ['c', 'd']
] 
FOR-EACH [i, j] 
IN $tabs
BEGIN
```
now is more readable